components/input/input.css

```diff
@@ -0,0 +1,115 @@
+form {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  padding: 2rem 2.4rem;
+  border-radius: 8px;
+  gap: 0.8rem;
+  background-color: var(--white);
+}
+
+form label {
+  font-size: 12px;
+  font-weight: 600;
+}
+
+form input {
+  padding: 8px 12px;
+  border: 1px solid gray;
+  border-radius: 2px;
+}
+
+form input:focus {
+  outline: none;
+}
+
+.form-checkbox {
+  display: flex;
+  align-items: center;
+  margin-top: 0.4rem;
+}
+
+.form-checkbox input {
+  height: 1rem;
+  width: 1rem;
+  margin-right: 6px;
+}
+
+.form-checkbox p {
+  font-size: 12px;
+  color: var(--gray);
+}
+
+.form-buttons {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 0.4rem;
+}
+
+::placeholder {
+  color: var(--gray);
+}
+
+.form-input-success {
+  color: var(--success-color);
+  display: flex;
+  flex-direction: column;
+  gap: 0.3rem;
+}
+.form-input-success input {
+  color: var(--success-color);
+  border: 1.6px solid var(--success-color);
+}
+.form-input-success input:focus {
+  outline: none !important;
```

Comment:
We should try to avoid using ! important as much as possible. We should use it only if there is no way around it.